### ISSUE TITLE
Settings file example: Insufficient contrast

### ISSUE THREAD

**Issue by dmethvin-gov** [uswds/uswds-site#1590]
## Description

The settings file example in docs would generate a warning for insufficient contrast, if the warnings hadn't been disabled. Since it's typical for people getting started to copy/paste examples, it would be better to have an initial file that didn't disable warnings and had contrasting colors.

## Steps to reproduce the issue

* Copy the [settings file example](https://github.com/uswds/uswds-site/blob/470517b5fd70fc3a4a041cddb08652a7128c19e7/pages/documentation/getting-started-developers/phase-two.md?plain=1#L61-L75)
* Change `$theme-show-compile-warnings` to `false`
* Run `npx gulp compile`
* Note the following warnings:
<summary>SASS Warnings</summary>
<details>
Warning: [Alert (success)] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `success-lighter` background. Using `primary`. Please check your source code and project settings.
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 59:7   get-color-token-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-link-tokens-from-bg.scss 79:27  get-link-tokens-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/set-link-from-bg.scss 18:17          set-link-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss 60:5        alert-status-styles()
    node_modules/@uswds/uswds/packages/usa-alert/src/styles/_usa-alert.scss 47:5                                 @forward
    usa-alert/src/_index.scss 4:1                                                                                @forward
    _index.scss 15:1                                                                                             @forward
    sass/styles.scss 2:1                                                                                         root stylesheet

Warning: [Alert (warning)] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `warning-lighter` background. Using `primary`. Please check your source code and project settings.
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 59:7   get-color-token-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-link-tokens-from-bg.scss 79:27  get-link-tokens-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/set-link-from-bg.scss 18:17          set-link-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss 60:5        alert-status-styles()
    node_modules/@uswds/uswds/packages/usa-alert/src/styles/_usa-alert.scss 47:5                                 @forward
    usa-alert/src/_index.scss 4:1                                                                                @forward
    _index.scss 15:1                                                                                             @forward
    sass/styles.scss 2:1                                                                                         root stylesheet

Warning: [Alert (error)] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `error-lighter` background. Using `primary`. Please check your source code and project settings.
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 59:7   get-color-token-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-link-tokens-from-bg.scss 79:27  get-link-tokens-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/set-link-from-bg.scss 18:17          set-link-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss 60:5        alert-status-styles()
    node_modules/@uswds/uswds/packages/usa-alert/src/styles/_usa-alert.scss 47:5                                 @forward
    usa-alert/src/_index.scss 4:1                                                                                @forward
    _index.scss 15:1                                                                                             @forward
    sass/styles.scss 2:1                                                                                         root stylesheet

Warning: [Alert (info)] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `info-lighter` background. Using `primary`. Please check your source code and project settings.
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 59:7   get-color-token-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-link-tokens-from-bg.scss 79:27  get-link-tokens-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/set-link-from-bg.scss 18:17          set-link-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss 60:5        alert-status-styles()
    node_modules/@uswds/uswds/packages/usa-alert/src/styles/_usa-alert.scss 47:5                                 @forward
    usa-alert/src/_index.scss 4:1                                                                                @forward
    _index.scss 15:1                                                                                             @forward
    sass/styles.scss 2:1                                                                                         root stylesheet

Warning: [Alert (info)] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `info-lighter` background. Using `primary`. Please check your source code and project settings.
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 59:7   get-color-token-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-link-tokens-from-bg.scss 79:27  get-link-tokens-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/set-link-from-bg.scss 18:17          set-link-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/alert-status-styles.scss 60:5        alert-status-styles()
    node_modules/@uswds/uswds/packages/usa-site-alert/src/styles/_usa-site-alert.scss 46:7                       @forward
    usa-site-alert/src/_index.scss 4:1                                                                           @forward
    _index.scss 42:1                                                                                             @forward
    sass/styles.scss 2:1                                                                                         root stylesheet

Warning: [Summary Box] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `info-lighter` background. Using `primary`. Please check your source code and project settings.
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 59:7   get-color-token-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-link-tokens-from-bg.scss 79:27  get-link-tokens-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/mixins/helpers/set-link-from-bg.scss 18:17          set-link-from-bg()
    node_modules/@uswds/uswds/packages/usa-summary-box/src/styles/_usa-summary-box.scss 48:3                     @forward
    usa-summary-box/src/_index.scss 4:1                                                                          @forward
    _index.scss 45:1                                                                                             @forward
    sass/styles.scss 2:1                                                                                         root stylesheet

Warning: [Summary Box] Neither the specified preferred link color token (`base-lighter`) nor the fallback link color token (`primary`) have AA contrast on a `info-lighter` background. Using `primary`. Please check your source code and project settings.
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-color-token-from-bg.scss 59:7   get-color-token-from-bg()
    node_modules/@uswds/uswds/packages/uswds-core/src/styles/functions/color/get-link-tokens-from-bg.scss 79:27  get-link-tokens-from-bg()
    node_modules/@uswds/uswds/packages/usa-summary-box/src/styles/_usa-summary-box.scss 54:19                    @forward
    usa-summary-box/src/_index.scss 4:1                                                                          @forward
    _index.scss 45:1                                                                                             @forward
    sass/styles.scss 2:1                                                                                         root stylesheet
</details>

## Additional information

I'd prefer a simple paste-able example here that retained default colors and didn't throw warnings, something like this:
```sass
@use "uswds-core" with (
  $theme-image-path: "../uswds/img",
  $theme-show-compile-warnings: true,
  $theme-show-notifications: false,
  $theme-banner-link-color: "primary-light",
  $theme-banner-max-width: "none",
)
```

LMK if you want a pull request for this. I really like 3.0, looking forward to using it on some real projects!
